### ISSUE TITLE
External LB examples are failing because of backend Schema change

### ISSUE THREAD

**Issue by ocsig** [GoogleCloudPlatform/cloud-foundation-toolkit#361]
```
[ { keyword: 'oneOf',
    dataPath: '.backends[0]',
    schemaPath: '#/properties/backends/items/oneOf',
    params: { passingSchemas: [Array] },
    message: 'should match exactly one schema in oneOf' } ]
```

Backend service schema requires `balancingMode`.

https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/blob/20d235a4636fc502c5534eb51cf81f43301d3422/dm/templates/external_load_balancer/examples/external_load_balancer_http.yaml#L26

**Comment by ocsig** [comment]
All 4 examples needs an update

**Comment by aaron-lane** [comment]
Fixed by #392.
